tests/build_tool.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------
+# PURPOSE
+
+# This is the build script for eris-db. It will build the
+# tool into docker containers in a reliable and predicatable
+# manner.
+
+# ----------------------------------------------------------
+# REQUIREMENTS
+
+# docker installed locally
+
+# ----------------------------------------------------------
+# USAGE
+
+# build_tool.sh
+
+# ----------------------------------------------------------
+# Set defaults
+set -e
+start=`pwd`
+if [ "$CIRCLE_BRANCH" ]
+then
+  repo=`pwd`
+else
+  base=github.com/eris-ltd/eris-db
+  repo=$GOPATH/src/$base
+fi
+branch=${CIRCLE_BRANCH:=master}
+branch=${branch/-/_}
+
+release_min=$(cat $repo/version/version.go | tail -n 1 | cut -d \  -f 4 | tr -d '"')
```

Comment:
good comment, but this should be done across the stack for consistency reasons and there is an argument for using solid bash. definitely worth raising an issue and following up